contexts/importer.ts

```diff
@@ -22,8 +22,17 @@ export const useConfigFileImporter = () => {
             notifications?: Notification[];
           } = JSON.parse(String(reader.result));
 
-          if (probes) handleSetProbes(probes);
-          if (notifications) handleSetNotifications(notifications);
+          if (probes) {
+            handleSetProbes(probes);
+          } else {
+            throw new Error('something wrong in probes key');
+          }
+
+          if (notifications) {
+            handleSetNotifications(notifications);
+          } else {
+            throw new Error('something wrong in notifications key');
```

Comment:
The notifications field is not a mandatory for monika, so I think this line should not throw error

reference: https://github.com/hyperjumptech/monika/blob/main/src/interfaces/config.ts

Reply:
fixed